packages/@dcl/legacy-ecs/src/decentraland/Components.ts

```diff
@@ -175,6 +176,38 @@ export class Transform extends ObservableComponent {
   }
 }
 
+/** @public */
+export enum AttachToAvatarAnchorPointId {
+  NameTag = 0,
+  LeftHand,
+  RightHand
+}
+
+/** @public */
+export type AttachToAvatarConstructorArgs = {
+  avatarId?: string
+  anchorPointId?: AttachToAvatarAnchorPointId
+}
+
+/**
+ * @public
+ */
+@Component('engine.transform', CLASS_ID.AVATAR_ATTACH)
```

Comment:
Using `engine.transform` means that an ECS entity cannot have both `Transform` and `AttachToAvatar` components. Just like an entity cannot have both `PlaneShape` and `BoxShape`. Is that intended?

Reply:
Yeap, it's intended. I think it was actually your idea when we talk about this a while ago :P
It makes sense cause it will be always easier to set something as a child of the attached entity and transform that, making transform local to the attached bone

Reply:
👍